redfish-core/lib/pcie_slots.hpp

```diff
@@ -29,6 +29,56 @@
 namespace redfish
 {
 
+inline void
+    linkAssociatedProcessor(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                            const std::string& pcieSlotPath, size_t index)
+{
+    dbus::utility::getAssociationEndPoints(
+        pcieSlotPath + "/connected_to",
+        [asyncResp, pcieSlotPath,
+         index](const boost::system::error_code& ec,
+                const dbus::utility::MapperEndPoints& endpoints) {
+        if (ec)
+        {
+            if (ec.value() == EBADR)
+            {
+                // This PCIeSlot have no processor association.
+                BMCWEB_LOG_DEBUG("No processor association found");
+                return;
+            }
+            BMCWEB_LOG_ERROR("DBUS response error",ec.message());
+            messages::internalError(asyncResp->res);
+            return;
+        }
+
+        if (endpoints.empty())
+        {
+            BMCWEB_LOG_DEBUG("No association found for processor");
```

Comment:
This would be `BMCWEB_LOG_ERROR()`

Reply:
Yes, please can this to LOG_ERROR. We sure this should be an internal error?